persistence.js

```diff
@@ -71,10 +76,13 @@ RedisPersistence.prototype.storeRetained = function (packet, cb) {
   }
 }
 
-RedisPersistence.prototype.createRetainedStream = function (pattern) {
+RedisPersistence.prototype.createRetainedStreamCombi = function (patterns) {
   var that = this
   var qlobber = new Qlobber(qlobberOpts)
-  qlobber.add(pattern, true)
+
+  patterns.map(function (pattern) {
+    qlobber.add(pattern, true)
+  })
```

Comment:
you should use a for loop here, it's way faster is this is going to be in a hot path.